Dockerfile

```diff
@@ -23,4 +23,6 @@ RUN poetry install
 # https://docs.docker.com/build/building/cache/#order-your-layers
 COPY . .
 
+WORKDIR /app/apps
```

Comment:
Looks like we could copy only `apps/` dir into docker image. In that case `COPY` command has to be updated instead of adding this line.

Changing WORKDIR of docker image to project subfolder is not flexible approach, because if we decide to run other commands in same docker container (like tests, linters, etc), the project root with config files (e. g. pytest.ini) won't be available for them.

We can consider modifying `PYTHONPATH` or `CMD` instead of `WORKDIR`